internal/terraform/rootmodule/root_module.go

```diff
@@ -22,9 +23,29 @@ import (
 	"github.com/hashicorp/terraform-ls/internal/filesystem"
 	"github.com/hashicorp/terraform-ls/internal/terraform/discovery"
 	"github.com/hashicorp/terraform-ls/internal/terraform/exec"
+	preloadedSchemas "github.com/hashicorp/terraform-ls/schemas"
 	tfschema "github.com/hashicorp/terraform-schema/schema"
 )
 
+var preloadedProviderSchemas tfjson.ProviderSchemas
+
+func init() {
```

Comment:
I wanted the schemas to be available as soon as possible, but json unmarshalling should be so fast I can do it in MergedSchema()

Reply:
Yeah, I think it would also make it easier to test if we needed/wanted to and relatedly we'd avoid side effects in unrelated tests (where the code would run just because we import the package).